quickstarts/gigamon-appinsights/config.yml

```
@@ -0,0 +1,46 @@
+
+
+slug: gigamon-appinsights
+
+title: Gigamon Application Insights
+
+description: |
+  ## Deep Application Visibility
+  Analytics tools are only as smart as the data they receive. Gigamon Application Metadata Intelligence empowers 
+  New Relic with critical metadata attributes across thousands of business, consumer and IT applications and services. 
+  Get deep application visibility to rapidly pinpoint performance bottlenecks, quality issues and potential network security risks.
+
+summary: |
+  Powered by Gigamon's Application Metadata Intelligence (AMI), part of Application Intelligence, to help you monitor and manage complex digital initiatives.
+
+level: Verified
+
+authors:
+  - Gigamon
+
+keywords:
+  - gigamon
+  - application
+  - network
+  - AMI
+  - intelligence
+  - metadata
+  - siem
+
+documentation:
+  - name: Application Metadata Intelligence
+    url: www.gigamon.com/products/optimize-traffic/application-intelligence/application-metadata.html
```

The system isn't a huge fan of `.html` if you remove that, it should be fine with this link.
```suggestion
    url: www.gigamon.com/products/optimize-traffic/application-intelligence/application-metadata
```